opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1577,6 +1731,8 @@ namespace Opm
             }
             deferred_logger.debug(message);
         } else {
+            this->wellStatus_ = well_status_orig;
+            this->operability_status_ = operability_orig;            
             const std::string message = fmt::format("   Well {} did not converged in {} inner iterations ("
```

Comment:
Please make this message the same as for StandardWell (including the grammar fix).

Reply:
Done